bitnami/rclone/1/debian-11/docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '2'
 services:
+  # [TEST]
```

Comment:
Is this supposed to be here?

Reply:
It is a trigger to run the VIB pipeline. It will be removed before merging the PR